### PR TITLE
Display FPU in short configuration

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**
image

### CODE REVIEW

**Comment by cquiroz** [comment]
Do we want to show it also as a column on the obs summary table?

**Comment by hugo-vrijswijk** [comment]
> Do we want to show it also as a column on the obs summary table?

It's also included there, I probably should've added the screenshot:

image

**Comment by cquiroz** [comment]
I see, I expected it to generate a new column but now I'm not sure if that's what we really want, maybe we need to confirm with @andrewwstephens, e.g. we may want to sort by fpu

**Review by toddburnside — COMMENTED** [review]
LGTM

**Review by toddburnside — APPROVED** [review]

File: model-tests/shared/src/test/scala/explore/model/ObsSummarySuite.scala [hunk 1]
```diff
@@ -14,4 +16,18 @@ class ObsSummarySuite extends DisciplineSuite {
            EqTests[ObsSummaryWithTitleConstraintsAndConf].eqv
   )
   checkAll("Eq[ObsSummaryWithTitleAndConf]", EqTests[ObsSummaryWithTitleAndConf].eqv)
+
+  property("ObsWithConf should include grating and fpu shortName") {
```
Comment (cquiroz) [inline model-tests/shared/src/test/scala/explore/model/ObsSummarySuite.scala:20]: Great to have more testts

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**